libkbfs/tlf_journal.go

```diff
@@ -883,97 +889,144 @@ func (j *tlfJournal) getJournalStatus() (TLFJournalStatus, error) {
 }
 
 func (j *tlfJournal) getJournalStatusWithRange() (
-	TLFJournalStatus, []ImmutableBareRootMetadata, bool, error) {
+	TLFJournalStatus, []ImmutableBareRootMetadata,
```

Comment:
Should probably then name and put the args in this order: `unflushedPaths unflushedPathsMap, ibrmds []ImmutableRootMetadata, complete bool`